### DISCUSSION TITLE
Initial viewing area

### DISCUSSION THREAD

**Discussion by oneart-dev** [reactivemarkets/react-financial-charts#504] [Q&A] [answered]
Hello, 
I'm so happy to find this library, just thank you!
Already implemented quite a few things and upgrades.
But I need help with initial view area.

Lets say I have 2 time values start and end. And I need to show all between this 2 time points when chart first open.

Can you help me where to look?

**Comment by markmcdowell** [comment, marked as answer]
At the moment, you can either set the xExtents to restrict the values or slice the data before you give it to the chart.

Most of the examples here use xExtents to just show the last 100 points, see https://github.com/reactivemarkets/react-financial-charts/blob/46f9274e64cd01d863e81416748888f8f96bb00f/packages/stories/src/features/StockChart.tsx#L75
